src/components/Header.js

```diff
@@ -3,9 +3,27 @@ import React from "react";
 const Header = () => {
   return (
     <div>
-      <h1> React and Morty </h1>
+      <HeaderStyles> React and Morty </HeaderStyles>
     </div>
```

Comment:
If I see this correctly you only need the `HeaderStyles` `h1 element`. Therefore, you don't need the `div` elements. :) You can also think about the naming of your h1 element. Maybe `Title` is better or `Heading` is better. 🤔